tests/testthat/test-tbl_split.R

```diff
@@ -1,9 +1,9 @@
 skip_on_cran()
```

Comment:
Can you add comments above each of the `expect_*()` function describing the test?